helpers/adapter.md

```diff
@@ -20,26 +20,40 @@ import { env, getRuntimeKey } from 'https://deno.land/x/hono/helper.ts'
 
 ## `env()`
 
-The `env()` function facilitates retrieving environment variables across different runtimes, extending beyond just Cloudflare Workers' Bindings.
+The `env()` function facilitates retrieving environment variables across different runtimes, extending beyond just Cloudflare Workers' Bindings. The value that can be retrieved with `env(c)` may be different for each runtimes.
 
 ```ts
 import { env } from 'hono/adapter'
 
 app.get('/env', (c) => {
+  // NAME is process.env.NAME on Node.js or Bun
+  // NAME is the value written in `wrangler.toml` on Cloudflare
   const { NAME } = env<{ NAME: string }>(c)
   return c.text(NAME)
 })
 ```
 
-Supported Runtimes:
+Supported Runtimes, Serverless Platforms and Cloud Services:
 
 - Cloudflare Workers
+  -  wrangler.toml
 - Deno
+  -  `Deno.env.set`
```

Comment:
How about just writing `Deno.env` and linking to this page? https://docs.deno.com/runtime/manual/basics/env_variables